src/components/jobManager.ts

```diff
@@ -330,7 +330,10 @@ export class JobManager {
 			if (
 				job &&
 				(job.kind === JobKind.rewriteFile ||
-					job.kind === JobKind.moveAndRewriteFile) &&
+					job.kind === JobKind.moveAndRewriteFile ||
```

Comment:
can you explain this change?

Reply:
This should clean all output files of the job from storage when the job is rejected